packages/odyssey-react-mui/src/labs/AppTile.tsx

```diff
@@ -169,7 +169,15 @@ const AppTile = ({
   );
 
   return (
-    <MuiCard className="isClickable">
+    <MuiCard
+      className="isClickable"
+      sx={{
+        boxShadow: "none",
+        "&.isClickable:hover": {
+          "box-shadow": odysseyDesignTokens.DepthMedium,
```

Comment:
Should this be `DepthHigh`? Like the other cards?

Reply:
we're trying to make this unique to the `AppTile`, not all `MuiCard`'s. `MuiCard` goes from medium -> high, and we want `AppTile` to go from nothing -> medium

Reply:
All good. Assuming design was ok with this then?

Reply:
yes https://okta.slack.com/archives/C7T2H3KNJ/p1727371369514169?thread_ts=1727303816.749929&cid=C7T2H3KNJ